vagrant/Vagrantfile

```diff
@@ -0,0 +1,53 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+
+  config.trigger.before :up do |trigger|
+      trigger.run = {inline: "cp -uvf ../../../../../bin/coredns ../../../../../bin/wgsd-client ."}
```

Comment:
on Darwin I get this error:

```
% vagrant up
Bringing machine 'registry' up with 'virtualbox' provider...
Bringing machine 'client-1' up with 'virtualbox' provider...
Bringing machine 'client-2' up with 'virtualbox' provider...
Bringing machine 'client-3' up with 'virtualbox' provider...
Bringing machine 'client-4' up with 'virtualbox' provider...
==> registry: Running action triggers before up ...
==> registry: Running trigger...
    registry: Running local: Inline script
    registry: cp -uvf ../../../../../bin/coredns ../../../../../bin/wgsd-client .
    registry: /bin/cp: illegal option -- u
```

I imagine this won't work unless the host is a Linux kernel. Instead of copying binaries from the host, can we build within setup.sh so that the Vagrantfile is portable across host OS? That also removes any cross compilation confusion.